app/controllers/katello/api/v2/ostree_branches_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Katello
+  class Api::V2::OstreeBranchesController < Api::V2::ApiController
+    apipie_concern_subst(:a_resource => N_("an ostree branch"), :resource => "ostree_branch")
+    include Katello::Concerns::Api::V2::RepositoryContentController
+
+    private
+
+    def resource_class
+      OstreeBranch
+    end
+
+    def filter_by_content_view_filter(filter)
```

Comment:
I'd probably re-word it to capture that the API is specifying content view filters as an API param on types that it does not apply to (https://github.com/Katello/katello/blob/f8509aea8c5fefd84c4f01037a551341f00293eb/app/controllers/katello/concerns/api/v2/repository_content_controller.rb#L25) more so than its a functional code issue. That being said, I think its a valid bug and I would just drop this function from your change since puppet modules suffer from the same fate.